data/json/monsters/insect_spider.json

```diff
@@ -2848,7 +2854,7 @@
     "name": { "str": "titan stag beetle" },
     "description": "This stag beetle rivals an armored personnel carrier in size, bearing horns that could fell a tree and chitin thick enough to ignore most bullets.  Its bulky shell looks nigh-impenetrable, and the cracks have sealed with a foul paste.",
     "copy-from": "mon_stag_beetle_giant",
-    "proportional": { "hp": 4, "speed": 0.75, "morale": 3, "melee_dice_sides": 2 },
+    "proportional": { "hp": 4, "speed": 0.75, "attack_cost": 0.75, "morale": 3, "melee_dice_sides": 2 },
```

Comment:
This thing is gigantinormous and pretty much invincible,, it doesn't need to put an attack out every turn to be scary.